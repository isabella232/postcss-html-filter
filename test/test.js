'use strict';

const fs = require('fs');
const path = require('path');
const cssSieve = require('..');

test('produces expected output', () => {
  const css = fs.readFileSync(path.join(__dirname, './fixtures/fixture.css'), 'utf8');
  const html = fs.readFileSync(path.join(__dirname, './fixtures/fixture.html'), 'utf8');
  return cssSieve.sift(css, html).then(output => {
    expect(output).toMatchSnapshot();
  });
});
