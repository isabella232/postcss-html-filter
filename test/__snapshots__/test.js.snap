// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`produces expected output 1`] = `
".one { color: pink; } /* keep */

@media (min-width: 80px) {
  .one { color: orange; } /* keep */
}

:first-letter { color: pink; }

body .three { background: black; } /* keep */
.two > .two-inner { color: yellow; } /* keep */
input { color: white; } /* keep */

[disabled] { color: gray; } /* keep */
*:not([disabled]) { color: orangutan; } /* keep */
#two[type=\\"text\\"] { display: none; } /* keep */
"
`;
